config/initializers/new_framework_defaults_7_2.rb

```
@@ -0,0 +1,70 @@
+# Be sure to restart your server when you modify this file.
```

rubocop (1.67.0)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)